frontend/src/Components/UserProfile.js

```diff
@@ -5,11 +5,16 @@ import { Loading } from "../Components/Common/Loader";
 import Post from "./Post/Post";
 import { toast } from "react-toastify";
 import { AuthContext } from "../Context/AuthContext";
+import EditProfile from "./Editprofile";
 import { A } from "hookrouter";
 
 export default function UserProfile({ id }) {
   const [userProfile, setUserProfile] = useState([]);
   const [loading, setLoading] = useState(false);
+  const [accessToken, setAccessToken] = useState(
+    localStorage.getItem("access_token")
+  );
```

Comment:
Instead of getting accessToken here, you can directly use the `AuthContext` to get access token as well as a user (if you want) back from it. See: https://github.com/MLH-Fellowship/SocioMark/blob/6c8857bbcc30a3ad9a228fbd29b5423a02aeec91/frontend/src/Components/UserFeed.js#L16-L17

And this: https://github.com/MLH-Fellowship/SocioMark/blob/6c8857bbcc30a3ad9a228fbd29b5423a02aeec91/frontend/src/Context/AuthContext.js#L27

Reply:
Doing `localStorage.getItem` at multiple places might be concerning as if we ever wanted to change the key name, we'd have to do it at multiple places.